syft/workers/base.py

```diff
@@ -453,6 +455,23 @@ def send(
 
         return pointer
 
+    def garbage(self, object_id, location):
+        """
+        Garbage manager which collects all the remote GC request and batch send
+        them every "delay" seconds for every location.
+        """
+        delay = self.object_store.garbage_delay
+        trash = self.object_store.trash
+
+        if location.id not in trash:
+            trash[location.id] = (time.time(), [])
+
+        trash[location.id][1].append(object_id)
+
+        if (time.time() - trash[location.id][0]) > delay:
```

Comment:
With the current `delay = 0`, this will always be the case and won't do any batching I guess

Reply:
I need a default delay = 0 to have the gc tests passing.
Yeah I could add an extra argument with the maximum capacity of the trash!